ads/_config.js

```diff
@@ -941,6 +941,8 @@ export const adConfig = {
     consentHandlingOverride: true,
   },
 
+  'torimochi': {},
```

Comment:
So I noticed here you are not implementing `renderStartImplemented`. If possible, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

Let me know what you think. Thank you! 😄